lib/decorate.js

```diff
@@ -34,7 +38,7 @@ function checkDependencies (instance, deps) {
 }
 
 function decorateReply (name, fn, dependencies) {
-  if (checkExistence(this._Reply, name)) {
+  if (checkExistenceInPrototype(this._Reply, name)) {
```

Comment:
let's just do `name in this._Reply`

Reply:
`name` is not in `this._Reply`. `name` is in its prototype.

If you want we could write `name in this._Reply.prototype`